src/System.Buffers.Experimental/System/Range.cs

```diff
@@ -1,26 +1,82 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace System {
-    public struct Range
+using System.Collections;
+using System.Collections.Generic;
+
+namespace System
+{
+
+    public struct Range : IEnumerable<int>
     {
-        public Range(long index, long length)
+        public const int UnboundedFirst = Int32.MinValue;
```

Comment:
@jaredpar - when we spoke yesterday, this was `static`.  Will `const` work as well?  Or does the compiler need it to be `static`?